src/shared/registered-app.component.js

```diff
@@ -0,0 +1,110 @@
+import React, { useContext } from "react";
+import { useCss } from "kremling";
+import Code from "./code.component";
+import { LocalStorageContext } from "./use-local-storage-data.hook";
+import { getPlaygroundDeps } from "../verify-app-guide/verification-steps/application-dependencies.component";
+
+export default function RegisteredApp({ app, edit, interactive }) {
+  const scope = useCss(css);
+
+  const { removeApplication } = useContext(LocalStorageContext);
+  const importMap = {
+    imports: {
+      [app.name]: window.importMapOverrides.getOverrideMap().imports[app.name],
+      ...sharedDepsImportMap(app.sharedDeps),
+    },
+  };
+  return (
+    <div {...scope}>
+      <div className="registered-app">
+        <Code code={indentedCode(app, importMap)} />
+        {interactive && (
+          <>
+            <button type="button" onClick={edit} className="edit">
+              Edit
+            </button>
+            <button type="button" onClick={removeButton} className="remove">
+              Remove
```

Comment:
Now that exactly one application is required, what does removing that application do? Does it put the user into an invalid state?

Reply:
Hmm, I think it doesn't really make sense... It doesn't put the user in a invalid state, because in places I need an application, I. added a check (like in root-config-guide page). Removing this feature, will definitely simply the code, should I?

Reply:
Yes, let's remove it